tests/NRedisStack.Tests/Search/IndexCreationTests.cs

```diff
@@ -0,0 +1,126 @@
+using StackExchange.Redis;
+using NRedisStack.Search;
+using NRedisStack.RedisStackCommands;
+using Xunit;
+using NRedisStack.Search.Literals;
+using NetTopologySuite.Geometries;
+
+namespace NRedisStack.Tests.Search;
+
+public class IndexCreationTests : AbstractNRedisStackTest, IDisposable
+{
+    private readonly string index = "MISSING_EMPTY_INDEX";
+    private static readonly string INDEXMISSING = "INDEXMISSING";
+    private static readonly string INDEXEMPTY = "INDEXEMPTY";
+
+    public IndexCreationTests(RedisFixture redisFixture) : base(redisFixture) { }
+
+    [SkipIfRedis(Comparison.LessThan, "7.3.240")]
+    public void TestMissingEmptyFieldCommandArgs()
```

Comment:
Just for clarification, as I see it's not an integration test, it doesn't execute a command against server. In this case what's the point to skip it on specific version?